visualizer/examples.js

```diff
@@ -26,13 +46,20 @@
 
   function checkExample(id) {
     var example = getExample(id);
+    var text = example.text;
+    var startRule = example.startRule;
+    var el = getListEl(id);
     var succeeded;
     try {
-      succeeded = ohmEditor.grammar.match(example).succeeded();
+      if (startRule) {
+        var matchResult = ohmEditor.grammar.match(text, startRule);
```

Comment:
You can just pass `startRule` unconditionally here -- passing null/undefined is the same as not passing it.